include/ofi_mr.h

```diff
@@ -227,6 +236,7 @@ struct ofi_mr_entry {
 	unsigned int			subscribed:1;
 	int				use_cnt;
 	struct dlist_entry		list_entry;
+	union ofi_mr_hmem_info		hmem_info;
```

Comment:
It looks like it would make sense to merge this patch with a modified version of the previous patch that adds the iface field to mr_entry rather than mr_info.

Reply:
Will do.